packages/react/src/components/Form/index.tsx

```diff
@@ -48,6 +48,10 @@ export interface FormFieldProps {
   fieldData: FormFieldData
   formState: any // eslint-disable-line @typescript-eslint/no-explicit-any
   fieldState: ControllerFieldState
+  /**
+   * Function for submitting the current step of the form.
+   */
+  submit: () => void
```

Comment:
Since this is being used for custom fields, what do you think about passing `handleSubmit` straight through from `useForm` then wrapping it with our primary handler in `NPSField`? Would allow implementors to wire up custom behaviors however they want, separate from the `onPrimary` handler.

Trade-off would be that folks need to know how to connect plug A to socket B when building custom fields, but if they're in there making custom form fields in the first place, that may be a fair requirement.

Reply:
I like it. Let me update it to do that. Much cleaner.

Reply:
Synced offline -- we will add this support in a future version.